packages/ui/src/icons/pending-icon.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+
+import { ISvgIcons } from "./type";
+
+export const PendingState: React.FC<ISvgIcons> = ({ width = "10", height = "11", className, color }) => (
```

Comment:
Set #455068 as the default color instead of hardcoding, so that we can override if required.